src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -77,39 +73,25 @@ public class ScriptTest {
     static final String sigProg = "47304402202b4da291cc39faf8433911988f9f49fc5c995812ca2f94db61468839c228c3e90220628bff3ff32ec95825092fa051cba28558a981fcf59ce184b14f2e215e69106701410414b38f4be3bb9fa0f4f32b74af07152b2f2f630bc02122a491137b6c523e46f18a0d5034418966f93dfc37cc3739ef7b2007213a302b7fba161557f4ad644a1c";
     static final String pubkeyProg = "76a91433e81a941e64cda12c6a299ed322ddbdd03f8d0e88ac";
 
-    private static final NetworkParameters PARAMS = TestNet3Params.get();
+    private static final NetworkParameters mainnetParams = MainNetParams.get();
     private static final Logger log = LoggerFactory.getLogger(ScriptTest.class);
-    private final List<BtcECKey> btcECKeyList = new ArrayList<>();
-    private final List<BtcECKey> erpFedECKeyList = new ArrayList<>();
-    private final BtcECKey ecKey1 = BtcECKey.fromPrivate(BigInteger.valueOf(100));
-    private final BtcECKey ecKey2 = BtcECKey.fromPrivate(BigInteger.valueOf(200));
-    private final BtcECKey ecKey3 = BtcECKey.fromPrivate(BigInteger.valueOf(300));
-    private final BtcECKey ecKey4 = BtcECKey.fromPrivate(BigInteger.valueOf(400));
-    private final BtcECKey ecKey5 = BtcECKey.fromPrivate(BigInteger.valueOf(500));
-    private final BtcECKey ecKey6 = BtcECKey.fromPrivate(BigInteger.valueOf(600));
-    private final BtcECKey ecKey7 = BtcECKey.fromPrivate(BigInteger.valueOf(700));
-    private final BtcECKey ecKey8 = BtcECKey.fromPrivate(BigInteger.valueOf(800));
-
-    @Before
-    public void setUp() {
-        btcECKeyList.add(ecKey1);
-        btcECKeyList.add(ecKey2);
-        btcECKeyList.add(ecKey3);
-        erpFedECKeyList.add(ecKey4);
-        erpFedECKeyList.add(ecKey5);
-        erpFedECKeyList.add(ecKey6);
-        erpFedECKeyList.add(ecKey7);
-        erpFedECKeyList.add(ecKey8);
-    }
+    private final List<BtcECKey> federationKeys = RedeemScriptUtils.getDefaultRedeemScriptKeys();
+    private final int expectedNumberOfRequiredSignatures = federationKeys.size() / 2 + 1;
+
+    private final List<BtcECKey> erpFederationKeys = RedeemScriptUtils.getEmergencyRedeemScriptKeys();
+    private final BtcECKey fedKey1 = federationKeys.get(0);
+    private final BtcECKey fedKey2 = federationKeys.get(1);
+    private final Address userAddress = BtcECKey.fromPrivate(BigInteger.valueOf(900)).toAddress(
```

Comment:
Same here, looks like this is only used in a couple of places

Reply:
Moved as well.